src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusInvalid.kt

```diff
@@ -59,21 +59,20 @@ suspend fun handleStatusINVALID(
     log.info("Apprec Receipt sent to {}, {}", apprecQueueName, fields(loggingMeta))
 }
 
-fun handleDuplicateSM2013Content(
+fun handleDuplicateDialogmeldingContent(
     session: Session,
     receiptProducer: MessageProducer,
     fellesformat: XMLEIFellesformat,
     loggingMeta: LoggingMeta,
     env: Environment,
-    redisSha256String: String
+    sha256String: String,
 ) {
-
     log.warn(
-        "Duplicate message: Same redisSha256String {}",
+        "Duplicate message: Same sha256String {}",
         createLogEntry(
             LogType.INVALID_MESSAGE,
             loggingMeta,
-            "originalEdiLoggId" to redisSha256String,
+            "shaString" to sha256String,
```

Comment:
Det er verdt å merke seg at til tross for navnet `redisSha256String`, var det faktisk ediLoggIDen som ble logget her. Derfor vil det se litt annerledes ut nå, siden jeg logger den faktiske sha-strengen.

Reply:
Det forklarer hvorfor det var ediloggiden jeg så i loggen :)